src/gui/element.rs

```diff
@@ -172,7 +217,9 @@ pub fn settings_container<'a>(
         .push(right_spacer);
 
     // Returns the final container.
-    Container::new(row).height(Length::Units(130))
+    Container::new(row)
+        .height(Length::Shrink)
+        .style(style::AddonRowDefaultTextContainer(color_palette))
```

Comment:
I changed the entire container to Shrink and then gave the right column a fixed height. This renders the settings page better when the width is reduced and the text starts wrapping, making the left column extend. Previously, if you resized the width down, the left column "Theme" button would extend the settings container and overlay the addons table.